### ISSUE TITLE
Wallpaper 

### ISSUE THREAD

**Issue by D-EVE-R** [8bitPit/Niagara-Issues#2373]
### Is your feature request related to a problem?

 No

### Describe the solution you'd like

Add daily wallpaper feature just like Olauncher thank you

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by tjsuominen** [comment]
There are 10's if not 100's of wallpaper apps available in play store, both paid and free. How would that add value for Niagara?

Or what are you referring to?

**Comment by D-EVE-R** [comment]
Yeah I agree with your suggestion but in my opinion x in 1 not like multi (one place ain't like multi places).
And also not every device capable of handling background processes you know what I mean lack of ram.
